### ISSUE TITLE
 get_logged_in_athlete_activities(self, after=0, list_activities=None)

### ISSUE THREAD

**Issue by pedrojunqueira** [sladkovm/stravaio#4]
Hi @sladkovm 

Thanks for putting up this package. This is the only one I am able to use given my level of python knowledge.

If I wanted to use the method  get_logged_in_athlete_activities() to pull activity summary of more than 30 activity. How can I return a list of activities with all the  my activities.

If I put no after parameter that would return more than 30 activity it only return a list of the last n activities that is the reminder of the division of total activities by 30.

Thanks

Pedro

**Comment by pedrojunqueira** [comment]
Hi @sladkovm 

I achieved what I needed using the following code.

Still need some refinement and error handling but here it goes

```
list_activities = []
after = timestamp
page = 1
while page <= 14:
    _fetched = activities_api.get_logged_in_athlete_activities(after=after, page=page, per_page=100)
    list_activities.extend(_fetched)
    print(f"Fetched {len(_fetched)}, between {_fetched[0].start_date} and {_fetched[-1].start_date}")
    page += 1
    if len(_fetched) == 0:
        break
```

**Comment by sladkovm** [comment]
Hi,

the intent is to fetch paginated output into a single list, but it indeed does not seem to work. I will look into it in the next few days.

Thanks for spotting the bug.

**Comment by sladkovm** [comment]
Fixed it - update to the version 0.0.9. 

If I do the command below I get a list of 170 activities summaries. 

```python
activities = client.get_logged_in_athlete_activities(after='last year')
```

I've also created an example notebook, but currently, GitHub has issues with rendering it [Jupyter Notebook](https://github.com/sladkovm/stravaio/blob/master/examples/fetch%20data%20example.ipynb)
